typing/datarepr.ml

```diff
@@ -154,7 +154,7 @@ let extension_descr ~current_unit path_ext ext =
   in
   let existentials, cstr_args, cstr_inlined =
     constructor_args ~current_unit ext.ext_private ext.ext_args ext.ext_ret_type
-      path_ext (Record_extension path_ext)
+      Path.(Pextra_ty (path_ext, Pext_ty)) (Record_extension path_ext)
```

Comment:
I find it weird that we would pass the same path before, and after the change we are not referring to the same objects anymore. What is the path exepected by the `Record_extension` constructor? (The path of the constructor, or the path of the inline record type?)

Reply:
To be precise: I wish the `Record_extension` constructor would document what path it expects.

Reply:
Okay, will fix it.

Reply:
Reflected at #11568